gulpfile.js/tests/configGenerator.test.js

```diff
@@ -0,0 +1,123 @@
+var fs = require('fs')
+var test = require('tape')
+var configGenerator = require('./../util/backstop/configGenerator')
+var deepClone = require('./../util/deepClone')
+var testFixturesPath = './gulpfile.js/tests/fixtures/configGenerator/'
+var config = {
+  compLib: {
+    port: 7000,
+    host: 'http://example',
+    baseDir: testFixturesPath
+  },
+  vrt: {
+    backstopConfigTemplate: testFixturesPath + 'backstop.template.json',
+    backstopConfig: testFixturesPath + 'backstop.json'
+  }
+}
+
+var removeFiles = function (files) {
+  return new Promise(function (resolve) {
+    files.forEach(function (file) {
+      fs.unlink(file, resolve)
+    })
+  })
+}
+
+var getBackstopConfigData = function () {
+  return new Promise(function (resolve, reject) {
+    fs.readFile(config.vrt.backstopConfig, function (err, data) {
```

Comment:
Gotcha. Maybe it feels like `configGenerator` is doing too much then. I was expecting `configGenerator` function to generate the config JSON and pass it to another function that writes it to a file. Got confused